### DISCUSSION TITLE
Load LitElement as ES6 module in browser without bundlers

### DISCUSSION THREAD

**Discussion by AlexeyBoiko** [lit/lit#2521] [General]
Hello, 
How to use Lit without bundlers? Is it possible to load LitElement as ES6 module in browser?

Here is web-component test.js file
```js
import { LitElement } from "../node_modules/lit-element/lit-element.js";
import { html } from "../node_modules/lit-html/lit-html.js";

export class SimpleGreeting extends LitElement {

  static properties = {
    name: {type: String},
  };

  constructor() {
    super();
    this.name = 'Somebody';
  }

  render() {
    return html`<p>Hello, ${this.name}!</p>`;
  }
}
customElements.define('simple-greeting', SimpleGreeting);
```

Here is html page
```html
<!doctype html>
<html>

<head>
	<script type="module" src="test.js"></script>
</head>
<body>
	<simple-greeting name="World"></simple-greeting>
</body>

</html>
```

In browser I get error
```
Failed to resolve module specifier "@lit/reactive-element". Relative references must start with either "/", "./", or "../".
```

**Comment by bicknellr** [comment]
Hi, I'm working on adding pre-built bundles right now. https://github.com/lit/lit.dev/pull/673 adds a section to the docs explaining how to use them and I'll merge it once the bundles are ready, so you can follow progress on this there. There's also an existing issue on this at https://github.com/lit/lit/issues/2272 .

**Comment by AlexeyBoiko** [comment]
@bicknellr , thank you for the information.
Lit documentation [Local dev servers](https://lit.dev/docs/tools/development/#devserver) helped to fix the problem.

>There are many development servers that can deal with module specifiers. If you already have a dev server that does this and integrates with your build process, that should be sufficient.
If you need a dev server, we recommend [Web Dev Server](https://modern-web.dev/docs/dev-server/overview/).

For this folder structure
```
proj
- node_modules
- src
- - test.js
- - index.html
web-dev-server.config.js
```

Works this web-dev-server.config.js
```js
export default {
	open: true,
	watch: true,
	appIndex: 'index.html',
	nodeResolve: true,
	esbuildTarget: 'auto',
	rootDir: 'src'
};
```
package.json
```json
{
  "name": "...",
  "version": "1.0.0",
  "description": "",
  "main": "index.html",
  "scripts": {
    "start": "web-dev-server"
  },
  "author": "...",
  "license": "...",
  "type": "module",
  "devDependencies": {
    "@web/dev-server": "^0.1.29",
    "@web/dev-server-esbuild": "^0.2.16",
    "eslint": "^7.32.0",
    "eslint-config-standard": "^16.0.3",
    "eslint-plugin-import": "^2.25.3",
    "eslint-plugin-node": "^11.1.0",
    "eslint-plugin-promise": "^5.2.0",
    "rollup-plugin-terser": "^7.0.2"
  },
  "dependencies": {
    "lit": "^2.1.3"
  }
}
```

For production build: [rollup](https://rollupjs.org/guide/en/) or cdn.
